roles/ceph-common/templates/ceph.conf.j2

```diff
@@ -30,41 +30,36 @@ mon initial members = {% for host in groups[mon_group_name] %}
     {% endfor %}
 {% endif %}
 
-{% if not containerized_deployment and not containerized_deployment_with_kv %}
-{% if monitor_address_block %}
-mon host = {% for host in groups[mon_group_name] %}{{ hostvars[host]['ansible_all_ipv4_addresses'] | ipaddr(monitor_address_block) | first }}{% if not loop.last %},{% endif %}{% endfor %}
-{% elif groups[mon_group_name] is defined %}
-mon host = {% for host in groups[mon_group_name] %}
-             {% set address = hostvars[host]['monitor_address'] if hostvars[host]['monitor_address'] is defined else monitor_address %}
-             {% set interface = hostvars[host]['monitor_interface'] if hostvars[host]['monitor_interface'] is defined else monitor_interface %}
-             {% if interface != "interface" %}
-               {% for key in hostvars[host].keys() %}
-                 {% if hostvars[host][key]['macaddress'] is defined and hostvars[host][key]['device'] is defined and hostvars[host][key]['device'] == interface -%}
-                    {{ hostvars[host][key][ip_version]['address'] }}
-                 {%- endif %}
-               {% endfor %}
-             {% elif address != "0.0.0.0" -%}
-               {{ address }}
-             {%- endif %}
-             {%- if not loop.last %},{% endif %}
-           {% endfor %}
-{% endif %}
-{% endif %}
+{% if not containerized_deployment and not containerized_deployment_with_kv -%}
+  mon host = {% for host in groups[mon_group_name] -%}
+  {% if hostvars[host]['monitor_address'] is defined and hostvars[host]['monitor_address'] != '0.0.0.0' -%}
```

Comment:
What if the given address is ipv6? Don't we need brackets?
Perhaps we should also check for ipv4 or ipv6?